src/components/PageNavLink/PageNavLink.tsx

```diff
@@ -0,0 +1,20 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+type Props = {
+  to: string,
+  text: string,
+};
+
+export const PageNavLink: React.FC<Props> = ({ to, text }) => (
+  <NavLink
+    to={to}
+    className={({ isActive }) => classNames('navbar-item',
+      {
+        'is-active': isActive,
+      })}
```

Comment:
you have class is-active in `li` element, here you can use `<Link>` and remove `is-active` class